edit.js

```diff
@@ -1,44 +1,125 @@
+let company = document.getElementById("company");
+let startDate = document.getElementById("start-date");
+let endDate = document.getElementById("end-date");
+let description = document.getElementById("description");
+
+let isEdit = false;
+
 function showExperienceForm() {
   document.getElementById("experience-form").style.display = "block";
 }
 document.getElementById("experience-form").addEventListener("submit", function (event) {
+  if (!company.value || !startDate.value || !endDate.value || !description.value) {
     event.preventDefault();
+    return;
+  }
+
+  event.preventDefault();
+  
+    let experiences = JSON.parse(localStorage.getItem("experience")) || [];
+
+    let id = Date.now().toString(36) + Math.random().toString(36).substr(2, 9);
+
+    if (isEdit) {
```

Comment:
is edit is being used as bolean at above and then used here as value, please recheck and fix